dl/tensorflow/matrix.yml

```diff
@@ -212,10 +218,11 @@ $render:
   # - 1.2.1
   # - 1.3.1
   # - 1.4.0
-  - 1.5.0
-  - 1.7.0
-  - 1.8.0
-  - 1.9.0
-  - '1.10'
-  - '1.11'
+  # - 1.5.0
```

Comment:
if there is a base update, shouldn't we rebuild all previous tf versions up to 1.5?

Reply:
Probably yes, but I'm a bit worried about the bug that we can introduce with this: there can be conflicts or incompatibilities issue of some sort between the framework and base packages (such as cuda 9.2 and TensorFlow 1.5). I think that it would be safer to update only the latest 2/3 releases. what do you think?